packages/language-kit/src/type.ts

```diff
@@ -0,0 +1,11 @@
+import { z } from 'zod';
+
+import { languages } from './const';
+
+export type LanguageTag = keyof typeof languages;
+
+export const isLanguageTag = (tag: string): tag is LanguageTag => tag in languages;
+
+export const languageTagGuard: z.ZodType<LanguageTag> = z
+  .any()
+  .refine((value: unknown) => typeof value === 'string' && isLanguageTag(value));
```

Comment:
nit: could move `typeof value === 'string'` in to the `isLanguageTag` method as well? Or rename the  `isLanguageTag` to be more specific? like isSupportedLanguageTag or so.

Reply:
I think we imply that they are "supported" in the language-kit package.
Therefore, no need to append "supported" prefix to them.